SCClassLibrary/Common/Collections/ObjectTable.sc

```diff
@@ -38,7 +38,29 @@ TwoWayIdentityDictionary : Collection
 		^objToID.at(obj);
 	}
 
+	copy {
+		^this.class.prNew(idToObj.copy, objToID.copy)
+	}
+
+	storeOn { arg stream;
+		if (stream.atLimit) { ^this };
+		stream << this.class.name << ".prNew(";
+		[idToObj, objToID].storeItemsOn(stream);
```

Comment:
yep, I think you are right, it dawned upon me later too.
Using ```add``` should be enough.